lms/djangoapps/shoppingcart/tests/test_models.py

```diff
@@ -254,13 +254,13 @@ def test_purchase(self):
             {
                 'orderId': 1,
                 'currency': 'usd',
-                'total': '40',
+                'total': '40.00',
```

Comment:
@nedbat Django 1.8 force decimal places for a `DecimalField`. Lets say we set `decimal_places=2` for a model `DecimalField` and then we set its value to `40` now upon retrieval we will get `40.00`. In Django 1.4 what we set is what we get and nothing extra.